test/e2e/specs/css.spec.js

```diff
@@ -0,0 +1,97 @@
+
+module.exports = {
+  'Basic CSS test' : function (client) {
+   var data = client.globals;
+    client
+      .url(client.launch_url)
+      .selectTab('CSS')
+      .setCssValue("body{background-color:#000}")
+        .frame(1)
+        .pause(data.defaultTime)
+        .frame(0)
+        .expect.element('body').to.have.css('backgroundColor').which.equals('rgba(0, 0, 0, 1)');
```

Comment:
That's where you would use custom asserions:
 .assert.outpuBgColor('rgba(0, 0, 0, 1)')